src/main/scala/io/github/kiemlicz/shelm/HelmPlugin.scala

```diff
@@ -306,34 +379,125 @@ object HelmPublishPlugin extends AutoPlugin {
       val ivy = ivySbt.value
       new ivy.Module(moduleSettings.value)
     },
-    publishConfiguration := PublishConfiguration()
-      .withResolverName(Classpaths.getPublishTo(publishTo.value).name)
-      .withArtifacts(packagedArtifacts.value.toVector)
-      .withChecksums(checksums.value.toVector)
-      .withOverwrite(isSnapshot.value)
-      .withLogging(UpdateLogging.DownloadOnly),
+
+    publishHelmToIvyRepo := true,
+    outstandingPublishRequests := 1, //careful
+    publishToHosting := Seq.empty,
+
+    publish := Def.sequential(
+      Def.taskIf {
+        if (publishHelmToIvyRepo.value) {
```

Comment:
TBH I don't see why you've introduced a flag for this particular `ChartHosting` subtype, but I have no problem with its existence

Reply:
Because previous version was misleading, logic was enabled given user provided element in `publishToHosting` of `IvyCompatibleHttpChartRepository` type  
Element was of no further use - hence I think it was misleading  

Because of that I think that the simplest flag will do just fine